src/main/java/core/basesyntax/vehicles/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.vehicles;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The Truck is working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The Truck is stopped");
```

Comment:
Similarly, the message in the `stopWork` method should be more informative. Consider updating it to 'Truck stopped its work.' to clearly indicate the action being performed by the specific type of machine.